hubspot-snowflake-aws-ml-insights/email-templates/AWS Personalize - Product Recommendations - Email HTML Module.module.html

```diff
@@ -0,0 +1,45 @@
+{% if is_in_hs_app %}
+  {% set contact_id = module.contact.properties.hs_object_id %}
+{% else %}
+  {% set contact_id = contact.hs_object_id %}
+{% endif %}
+
+{% if contact_id %}
+  {% set associated_recommendations = crm_associations(contact_id, "USER_DEFINED", 18, "orderBy=-hs_createdate&limit=5", "recommended_product_id") %}
```

Comment:
Suggestion: we may want a code comment and/or readme that describes 1) what object type we're associating to (product recommendations) and even 2) how you get the ID to reference (via the associations schema API)